docs/appendix.rst

```diff
@@ -86,6 +86,21 @@ This documentation uses a few special terms to refer to Python types:
    CIM object
       one of the types listed in :ref:`CIM objects`.
 
+   CIM namespace
+      an object that is accessible through a WBEM server and is a naming
+      space for CIM classes, CIM instances and CIM qualifier declarations.
+      The namespace is a component of other elements like namespace path used
+      to access objects in the WBEM server.
+
+   interop namespace
+      A :term:`CIM namespace` that must have one of the following names
+      ('interop', 'root/interop') and which contains components of a CIM model
+      that the client needs to discover the model and registered profiles
+      supplied by that server.  This namespace normally includes model
+      components to discover information on the WBEM server itself, its CIM
+      namespaces, and the registered profiles it implements.  A server must
+      have only a single interop namespace.
```

Comment:
It would be helpful to state which interop namespaces pywbem recognizes.

Reply:
DONE